### PR TITLE
Docs: add Lightbox global capability and proportioning based on viewport

### PR DESCRIPTION
**Pull request by michaellisitsa**
- [Add] Docs for Lightbox
- [Add] Docs for using media queries to proportion text for lightbox's large viewports
- [Add] Viewport specific font-sizing in static diagram example

### CODE REVIEW

File: docs/global-capabilities.md [hunk 1]
```diff
@@ -11,7 +11,39 @@ This feature is still in development.
...
+
+![Screenshot of lightbox option on diagram](_media/global-capabilities/global-capabilities-lightbox-button.png ":size=400")
+
+</div>
+
+Full screen active:
+
+<div style="text-align: center;">
+
+![Screenshot of lightbox activated](_media/global-capabilities/global-capabilities-lightbox-active.png ":size=600")
```
Comment (JulianGoh5) [inline docs/global-capabilities.md:28]: Noticed that the image attached doesn't show the zoom/fullscreen buttons?

Comment (michaellisitsa) [reply docs/global-capabilities.md:28]: The point of this image is to show the full screen rendering of the starter example, particularly around the relative text size to the shapes. I don't disagree that we could show them, but just don't see it as critical given the previous image shows the zoom buttons and is annotated "Toggle Full Screen" already.

**Comment by michaellisitsa** [comment]
@JulianGoh5 Clarified comment

File: docs/global-capabilities.md (hunk 1 above)
Comment (JulianGoh5) [reply docs/global-capabilities.md:28]: Without the controls it just looks like a random screenshot. When you're in fullscreen the controls are always shown at the bottom too, why exclude them it's intending to show that the "lightbox is active"?

Comment (michaellisitsa) [reply docs/global-capabilities.md:28]: Happy to add it in.

**Comment by michaellisitsa** [comment]
@JulianGoh5 Updated, including using full screen terminology per comment in PR https://github.com/ClearCalcs/custom-diagram-boilerplate/pull/50#discussion_r1675255289

File: docs/global-capabilities.md (hunk 1 above)
Comment (michaellisitsa) [reply docs/global-capabilities.md]: Updated here 0bb48edf5896b1c35609bbd2c3621ac737deb243

**Review by JulianGoh5 — APPROVED** [review]
Sweet, looks good

**Pull request merged**